config/muxcable.py

```diff
@@ -406,12 +406,13 @@ def state(state, port):
 
             if not isinstance(physical_port_list, list):
                 click.echo(("ERR: Unable to locate physical port information for {}".format(port)))
-                rc = False
-                if len(physical_port_list) != 1:
-                    click.echo(("ERR: Found multiple physical ports associated with {}, physical Ports:".format(port)))
-                    for lport in physical_port_list:
-                        click.echo(("{}".format(lport)))
-                    rc = False
+                continue
+
+            if len(physical_port_list) != 1:
+                click.echo(("ERR: Found multiple physical ports associated with {}, physical Ports:".format(port)))
+                for lport in physical_port_list:
+                    click.echo(("{}".format(lport)))
```

Comment:
You can omit the trailing newline by passing `nl=False` (see [here](https://click.palletsprojects.com/en/7.x/utils/)). Also, you have unnecessary double parentheses.

However, I suggest doing something like this instead:

```python
click.echo("ERR: Found multiple physical ports ({}) associated with {}".format(", ".join(physical_port_list), port))
```

Reply:
fixed